docs/apis-clients/operate-api/index.md

```diff
@@ -63,15 +58,13 @@ You will get something like the following:
 }
 ```
 
-Take the `access_token` value from the response object and store it as your token.
+Capture the `access_token` value from the response object. In each request to the Operate API, include it as an authorization header:
 
-2. Send the token as an authorization header in each request. In this case, request all process definitions.
-
-```shell
-curl -X POST 'http://localhost:8080/v1/process-definitions/search' -H 'Content-Type: application/json' -H 'Authorization: Bearer eyJhb...' -d '{}'
+```
+Authorization: Bearer eyJHb...
 ```
 
-#### Cookies
+#### Authentication via cookies
```

Comment:
🍪 This is a nitpick - should be singular cookie.

Reply:
Resolved in latest commit 👍